apps/src/sites/studio/pages/levelbuilder_pixelation.js

```diff
@@ -1,19 +1,52 @@
 /** @file Javascript for the pixelation widget levelbuilder edit page */
 import $ from 'jquery';
+import 'jquery-ui';
+
+let pixelationVersionDropdown;
+let hideSlidersField;
+let initialDimensionsField;
 
 $(() => {
-  const pixelationVersionDropdown = $('#level_version');
+  pixelationVersionDropdown = $('#level_version');
+  hideSlidersField = $('.hide-sliders');
+  initialDimensionsField = $('.initial-dimensions');
 
   // When the version changes, show/hide relevant fields
   pixelationVersionDropdown.change(() => {
     showAndHideFieldsForVersion(pixelationVersionDropdown.val());
   });
 
-  // On load, show/hide fields based on version
-  showAndHideFieldsForVersion(pixelationVersionDropdown.val());
+  // On load, show/hide fields based on version without animation
+  showAndHideFieldsForVersion(pixelationVersionDropdown.val(), false);
 });
 
-function showAndHideFieldsForVersion(version) {
-  // Only show the initial width/height fields for version 1
-  $('.initial-dimensions').toggle(version === '1');
+function showAndHideFieldsForVersion(version, animate=true) {
```

Comment:
I generally tend to avoid using default parameters. I'm not sure what the conditions are under which they are worth using. Thoughts?